src/index.js

```diff
@@ -481,4 +481,9 @@ export var app = function(props) {
   }
 
   dispatch(props.init)
+
+  return function () {
+    dispatch = function () {}
+    patchSubs(subs, [])
+  }
```

Comment:
probably it also should remove event listeners from DOM nodes
and probably it also should remove created DOM nodes
don't you think?

Reply:
I don't think kill should remove created dom-nodes because it's easy enough to do in userland, if you want -- and because there are cases where you *don't* want the created nodes removed. 

As for removing event listeners, `dispatch = function () {}` is effectively equivalent. We don't remove any event listeners, but all event handlers become no-ops. I don't think it would be worthwhile to explictly remove event listeners, as long as interacting with the dead DOM can't *do* anything.

Reply:
I don't think we also need to return the `node`, could you get rid of this? If the user wants the node, they can add an id and use `getElementById`.

Reply:
@jorgebucaran agreed, and done

Reply:
@frenzzy @zaceno Do you think this could work? 😄 

```js
patchSubs(subs, EMPTY_ARR, dispatch = Function)
```

Reply:
@jorgebucaran Yes I don't see why not! 🏌